packages/flutter_tools/lib/src/commands/fuchsia_reload.dart

```diff
@@ -90,43 +107,54 @@ class FuchsiaReloadCommand extends FlutterCommand {
 
     // Set up a device and hot runner and attach the hot runner to the first
     // vm service we found.
-    final int firstPort = targetPorts[0];
-    final String fullAddress = '$_address:$firstPort';
-    final FuchsiaDevice device = new FuchsiaDevice(fullAddress);
+    final List<String> fullAddresses =
+        targetPorts.map((int p) => '$_address:$p').toList();
```

Comment:
We prefer avoiding breaks after assignment operators, so that this would look like:
```dart
final List<String> fullAddresses = targetPorts.map(
  (int p) => '$_address:$p',
).toList();
```